samples/integration/vcpkg/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# base image installs keyvault beta1 from vcpkg only
+FROM vhvb1989/keyvault:beta1
+
+RUN cd vcpkg \
+    && sudo ./vcpkg install azure-identity-cpp
```

Comment:
Just this? I think we should use storage as an example instead.

Reply:
I can add another sample for storage.  why not.

Reply:
Added storage sample as well